README.md

```diff
@@ -65,6 +66,21 @@ Key																| Required	| Default					| Description
 `MYSQL_SERVICE_WAIT_INTERVAL`			| N					| 1000						| Number of milliseconds to wait between MySQL service uptime test retries
 `MYSQL_SERVICE_WAIT_MAX_RETRIES`	| N					| 30							| Maximum number of times to retry MySQL service uptime test before timing out
 
+### PostgreSQL
+
+Key																	| Required	| Default										| Description
+---																	| ---				| ---												| ---
+`POSTGRES_CONTAINER_NAME`						| Y         | _none_										| Name used to identify PostgreSQL Service Docker container
+`POSTGRES_DATABASE`									| Y         | _none_										| Name used to identify PostgreSQL Service database
+`POSTGRES_EXPOSED_PORT`							| Y         | 3306											| Local network port used to expose PostgreSQL Service
```

Comment:
If we want to keep it to the default port it should be 5432 for Postgres, 3306 is MySQL